### ISSUE TITLE
Improve accessibility of the autocomplete plugin (divarea editor only)

### ISSUE THREAD

**Issue by f1ames** [ckeditor/ckeditor4#4617]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

Extracted from #2199:

> Autocomplete accessibility markup should be reviewed. Items should gain proper roles etc.

Since there is no way to provide support in a reasonable way for `iframe` based editor (classic editor), see https://github.com/ckeditor/ckeditor4/issues/2199#issuecomment-818711934, we can cover this for `divaera` / `inline` one at least.
